python/lsst/display/matplotlib/matplotlib.py

```diff
@@ -208,9 +208,9 @@ def format_coord(x, y, wcs=self._wcs, x0=self._xy0[0], y0=self._xy0[1],
                 col -= x0
                 row -= y0
 
-                msg += ' %1.3f' % (self._image.get(col, row))
+                msg += ' %1.3f' % (self._image[col, row])
                 if self._mask:
-                    val = self._mask.get(col, row)
+                    val = self._mask[col, row]
```

Comment:
Not sure if it matters in this context (or if the previous version was even correct), but to make this identical to what it was before, you want `[col, row, afwImage.LOCAL]`.